autonomy/fsm/scaffold/base.py

```diff
@@ -100,11 +99,6 @@ def rounds(self) -> List[str]:
         """Non-degenerate rounds"""
         return sorted(self.dfa.states - self.dfa.final_states)
 
-    @property
-    def base_names(self) -> List[str]:
-        """Base names"""
-        return [s.replace(ROUND, "") for s in self.rounds]
-
```

Comment:
this was a bad method to begin with, since it makes assumptions that do not hold (that is that `Round` is always last part of the name of a round class, which it is not)